.gitlab-ci.yml

```diff
@@ -49,6 +49,7 @@ Data fetch:
         NOW=$(date +"%Y%m%d_%H%M")
         git add data
         git commit -m "feat(data): $NOW update"
+        git push --no-verify 
```

Comment:
does ` origin HEAD:master`  not needed ?

Reply:
yes, needed, updated since.

without it i got : 

```sh
fatal: You are not currently on a branch.
 To push the history leading to the current (detached HEAD)
 state now, use
     git push origin HEAD:<name-of-remote-branch>
```

Reply:
je soupconne le runner d'etre en detach head